deploy-board/deploy_board/webapp/helpers/environs_helper.py

```diff
@@ -89,14 +91,22 @@ def create_identifier_for_new_stage(request, env_name, stage_name):
         return None
 
     # retrieve Nimbus identifier for existing_stage
-    existing_stage_identifier = get_nimbus_identifier(request, stage_with_external_id['externalId'])
+    try:
+        existing_stage_identifier = get_nimbus_identifier(request, stage_with_external_id['externalId'])
+    except TeletraanException as detail:
+        log.error('Handling TeletraanException when trying to access nimbus API, error message {}'.format(detail))
+        messages.add_message(request, messages.ERROR, detail)
```

Comment:
This change will cause the function to continue execute. Is it desired?

Reply:
Yes, in the previous state if `get_nimbus_identifier` failed it would surface an error, but not render the page. We want to render the page with a message, which this accomplishes. I looked at the references to this method and it seems safe to allow it to execute

Reply:
Can you double check with @osoriano? He added 
`external_id = environs_helper.create_identifier_for_new_stage(request, name, stage)`

Reply:
This request is in the write path. I see it's used for creating new stages.

I believe the Nimbus id is a requirement for creating a new stage, so I don't think we could continue here. Otherwise, it would be possible to create a new stage without a Nimbus ID.

Same applies to the `create_nimbus_identifier` below

Reply:
Gotcha, the original ticket wanted to prevent Nimbus error from blocking certain pages from rendering, but this seems like maybe this can be the exception since it’s a write path vs rendering something like the environment landing page. @tylerwowen I will revert this change as well as the one on line 105